### DISCUSSION TITLE
211275CC-D79A-9C81-A2DF-6047DD4AC35D

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#58] [General]
# 211275CC-D79A-9C81-A2DF-6047DD4AC35D

Week 15, 2023: Europe's First Mission to Jupiter Successfully Launches!

https://celestia.mobi/article?guide=211275CC-D79A-9C81-A2DF-6047DD4AC35D&lang=zh_CN

**Comment by 13940951572** [comment]
这款卫星也不是4月14日发射的呀？

**Comment by bird0cosmic** [comment]
Ooo
